### PR TITLE
{devel} [intel-2015a] Likwid 3.1.2 (OK)

### PR DESCRIPTION
**Pull request by wpoely86**
There is an `pinomp-pthread-overload.patch` but it's unclear to me what it exactly does and if it is needed...

### CODE REVIEW

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux debian 7.8, Intel(R) Xeon(R) CPU            5130  @ 2.00GHz, Python 2.7.3
See https://gist.github.com/1eefaf4785db1003a23f for a full test report.

File: easybuild/easyconfigs/l/likwid/likwid-3.1.3-intel-2015a.eb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+description = """Likwid stands for Like I knew what I am doing. This project contributes easy to use
+ command line tools for Linux to support programmers in developing high performance multi threaded programs."""
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+
+sources = [SOURCE_TAR_GZ]
+source_urls = ['http://ftp.fau.de/pub/likwid/']
+
+skipsteps = ['configure']
+buildopts='CC="$CC" CFLAGS="$CFLAGS -std=c99"'
```
Comment (boegel) [inline easybuild/easyconfigs/l/likwid/likwid-3.1.3-intel-2015a.eb]: space around `=`

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/acaca85760fc7a7827dc for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**FAILED**
Build succeeded for 1 out of 2 (2 easyconfigs in this PR)
Linux debian 7.8, Intel(R) Xeon(R) CPU            5130  @ 2.00GHz, Python 2.7.3
See https://gist.github.com/cbe6d8f4919f94b8c061 for a full test report.

**Comment by boegel** [comment]
@wpoely86: better remove the `pinomp` version again, since the patch is broken?

We can always re-add the `pinomp` version later...

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 1 out of 2 (2 easyconfigs in this PR)
Linux SL 6.5, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/0bcd20f59228dc076416 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.5, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/968eac269a2cd68f84cc for a full test report.

**Comment by boegel** [comment]
going in, thanks @wpoely86!

**Pull request merged**